cdn-uploader.js

```diff
@@ -0,0 +1,41 @@
+const ftp = require("basic-ftp");
+
+const config = {
+	host: process.env.AKAMAI_CDN_HOSTNAME,
+	user: process.env.AKAMAI_CDN_USERNAME,
+	password: process.env.AKAMAI_CDN_PASSWORD,
+	cpCode: process.env.AKAMAI_CDN_CPCODE,
+	port: 21,
+}
+
+function UploadFilesFromPath(config) {
+	console.log("[+] Connecting to CDN...");
+	async function Upload() {
+		try {
+			const client = new ftp.Client();
+			await client.connect(config.host, config.port);
+			await client.login(config.user, config.password);
+			await client.useDefaultSettings();
+			console.log(`[+] Starting to upload the content of "${config.buildPath}".`);
+			await client.ensureDir(`/${config.cpCode}/${config.basePath}/${config.buildPath}`);
+			await client.clearWorkingDir();
+			await client.uploadDir(config.buildPath);
+			console.log(`[+] Folder "${config.buildPath}" uploaded.`);
+			client.close();
+		}
+		catch(err) {
+			console.log(err);
```

Comment:
What do you think about removing the catch? and break the publish if something goes wrong?